x-pack/elastic-agent/_meta/config/common.reference.p2.yml.tmpl

```diff
@@ -107,6 +107,9 @@ inputs:
 #   logs: false
 #   # enables metrics monitoring
 #   metrics: false
+#   # exposes /debug/pprof/ endpoints
+#   # recommended that these endpoints are only enabled if the monitoring endpoint is set to localhost
+#   pprof: true
```

Comment:
Why is `pprof: true` while `logs` and `metrics` are false by default?

Reply:
I think that the reference here may be incorrect, the default `MonitoringConfig` has them set to `true`